manager/middlewares/rbac.go

```diff
@@ -37,16 +42,22 @@ func isAccessGranted(c *gin.Context) bool {
 			status = res.StatusCode
 		}
 		serviceErrorCnt.WithLabelValues("rbac", strconv.Itoa(status)).Inc()
-		return false
+		return rbacPerms{Read: false, Write: false}
 	}
-	// For now we either allow access or don't allow it
+
 	for _, a := range access.Data {
-		if a.Permission == "patch:*:*" {
-			return true
+		switch a.Permission {
+		case "patch:*:*":
+			return rbacPerms{Read: true, Write: true}
+		case "patch:*:read":
+			return rbacPerms{Read: true, Write: false}
+		case "patch:*:write":
+			return rbacPerms{Read: false, Write: true}
```

Comment:
Write should probably imply read

Reply:
I check how other apps do it. Thay also use all *,read,write variants. So I used it too here.